doc/workshops/tutorial/content/intro/zapdos.md

```diff
@@ -0,0 +1,51 @@
+# Zapdos Introduction
+
+!---
+
+# Zapdos Overview
+
+- Zapdos is an open source multi-fluid plasma code
+
+  - Utilizes the finite element method.
+  - Solves for species densities in log-molar form.
+
+
+- As a MOOSE based application Zapdos can take advantage of:
+
+  - Scalable parallel computing.
+  - Flexible Multi-physics coupling.
+
+!---
+
+# Current Research Efforts style=margin-top:1em;
+
+- Validation and Verification
+
+  - Modeling physical systems and comparing to experimental results.
+  - Utilizing the Methods of Manufactored Solutions (MMS) to ensure correct implementation.
+
+!row!
+!col! small=12 medium=4 large=4 icon=flash_on
+!media media/experimental.png
+       style=width:60%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
+       caption=Experimental Comparison.
```

Comment:
Commenting since I didn't mention it on the original tutorial slides. For the plot on that slide, it was microwave interferometry measurements (K. E. Greenberg and G. A. Hebner, J. Appl. Phys. 73 (1993) 8126). For general experimental validation, we should probably reference the [V&V paper](https://www.sciencedirect.com/science/article/abs/pii/S0010465523001820).

Reply:
Added the citation to this page.